core/benchmarks/LZFBenchmark-jdk21-results.txt

```diff
@@ -0,0 +1,19 @@
+================================================================================================
+Benchmark LZFCompressionCodec
+================================================================================================
+
+OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1021-azure
+AMD EPYC 7763 64-Core Processor
+Compress small objects:                           Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+--------------------------------------------------------------------------------------------------------------------------------
+Compression 256000000 int values in parallel                598            600           2        428.2           2.3       1.0X
+Compression 256000000 int values single-threaded            568            570           2        451.0           2.2       1.1X
+
+OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1021-azure
+AMD EPYC 7763 64-Core Processor
+Compress large objects:                        Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+-----------------------------------------------------------------------------------------------------------------------------
+Compression 1024 array values in 1 threads                39             45           5          0.0       38475.4       1.0X
```

Comment:
With GitHub standard action runners, it seems that we only get 1 thread.

Reply:
hmm... 

```
[info] OpenJDK 64-Bit Server VM 17.0.10+7-LTS on Mac OS X 14.5
[info] Apple M2 Max
[info] Compress small objects:                           Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
[info] --------------------------------------------------------------------------------------------------------------------------------
[info] Compression 256000000 int values in parallel                548            550           2        467.0           2.1       1.0X
[info] Compression 256000000 int values single-threaded            522            523           1        490.5           2.0       1.1X
[info] Running benchmark: Compress large objects
[info]   Running case: Compression 1024 array values in 8 threads
[info]   Stopped after 123 iterations, 2009 ms
[info]   Running case: Compression 1024 array values single-threaded
[info]   Stopped after 83 iterations, 2003 ms
[info] OpenJDK 64-Bit Server VM 17.0.10+7-LTS on Mac OS X 14.5
[info] Apple M2 Max
[info] Compress large objects:                        Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
[info] -----------------------------------------------------------------------------------------------------------------------------
[info] Compression 1024 array values in 8 threads                12             16          13          0.1       11546.1       1.0X
[info] Compression 1024 array values single-threaded             23             24           1          0.0       22767.9       0.5X

```

I ran this benchmark locally, and it seems that the performance of `Compression 256000000 int values single-threaded` and `Compression 256000000 int values in parallel` is almost the same

Reply:
I guess the rate is limited by the producer